### DISCUSSION TITLE
教育网最近是不是在阻断cf ipv4

### DISCUSSION THREAD

**Discussion by jtzhpf** [XIU2/CloudflareSpeedTest#394] [讨论]
教育网下走CF IPv4的vmess全部timeout，走域名或者直接走IP走都不行，但是直接ping cf的IP是正常的，用curl -v --connect-to ::example.com: https://xx.xx.xx试了下握手也是正常的应该也不是sni阻断，浏览挂在cf上的网站也是没问题，请问大家还有可能是什么问题呢？

**Comment by charSLee013** [comment]
应该是vmess被阻断的问题了

测了一下用TCP去连接是没问题的，但是换成vmess 协议去连接显示**Timeout**

看了之前发布的 [GFW报告](https://gfw.report/publications/usenixsecurity23/zh/) ， 像使用 `TLS in TLS` 协议大部分已经被检测到了

**Reply by jtzhpf** [reply]
教育网可能不是这个原因，前几天我tcping cf的ip段，数个tcping包之后就开始丢包，而且丢包是连续的，一会之后又能ping通，丢包率差不多在80%左右，今天教育网已经恢复正常了。

**Reply by jtzhpf** [reply]
电信这几天也有问题，坐标合肥，不过电信应该就是你说的阻断vmess，v4完全没办法用，v6勉强能用用

**Reply by mengguyi** [reply]
北京电信也有断流，但没你得那么严重。主要发生在晚高峰时段

**Reply by charSLee013** [reply]
奇怪的是我用的机场是Vmess 直连国外IP的畅通无阻

反而自建VPS的Vemss 走Cloudflare CDN 通道不行

联想到前几天v2ex 论坛上有人可以让Cloudflare CDN被阻断 [罪大恶极我能 99%几率让使用 CloudFlare 的网站从中国大陆三网受屏蔽
](https://www.v2ex.com/t/949599#reply58)，可能是受此影响导致的

**Reply by jtzhpf** [reply]
可也不至于那么多cf ipv4都不行啊，我这边优选了100来个不同ip段的ip，电信网下tcping都没问题，丢包几乎不存在，上网页也没问题，但是vmess就是用不了，感觉应该是tls in tls检测
